firmware/quadruna/BMS/src/app/app_thermistors.c

```diff
@@ -28,10 +28,11 @@ void app_thermistors_updateAuxThermistorTemps()
 
 void app_thermistors_broadcast()
 {
-    app_canTx_BMS_ThermTemp0_set(therm_temps[0]);
-    app_canTx_BMS_ThermTemp1_set(therm_temps[1]);
-    app_canTx_BMS_ThermTemp2_set(therm_temps[2]);
-    app_canTx_BMS_ThermTemp3_set(therm_temps[3]);
-    app_canTx_BMS_ThermTemp4_set(therm_temps[4]);
-    app_canTx_BMS_ThermTemp5_set(therm_temps[5]);
+    // Re-enable if aux thermistors installed 
+    // app_canTx_BMS_ThermTemp0_set(therm_temps[0]);
+    // app_canTx_BMS_ThermTemp1_set(therm_temps[1]);
+    // app_canTx_BMS_ThermTemp2_set(therm_temps[2]);
+    // app_canTx_BMS_ThermTemp3_set(therm_temps[3]);
+    // app_canTx_BMS_ThermTemp4_set(therm_temps[4]);
+    // app_canTx_BMS_ThermTemp5_set(therm_temps[5]);
```

Comment:
Aux Thermistors not currently installed, no sense loading the CAN bus with their messages

Reply:
You should delete them from the JSON instead, commenting out the setters doesn't disable them from the bus. They'll just be transmitted with their start value (probably all zeros).

Reply:
Ah okay gotcha. Will do.